include/util/geojson_debug_logger.hpp

```diff
@@ -47,7 +47,7 @@ class GeojsonLogger
     // cannot lock, is tooling for locked function
     static void output(bool first, const util::json::Array &array)
     {
-        for (const auto object : array.values)
+        for (const auto &object : array.values)
```

Comment:
No longer copying potentially expensive variants around - this is the logger only so not really an issue